src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel.cpp

```diff
@@ -1570,6 +1570,26 @@ class at_icc_start_blood_quickening : public AreaTriggerScript
         }
 };
 
+class at_icc_nerubar_broodkeeper : public AreaTriggerScript
+{
+    public:
+        at_icc_nerubar_broodkeeper() : AreaTriggerScript("at_icc_nerubar_broodkeeper") { }
+
+        bool OnTrigger(Player* player, AreaTriggerEntry const* /*areaTrigger*/)
+        {
+            if (player->IsGameMaster())
+                return false;
+
+            std::list<Creature*> creatureList;
+            GetCreatureListWithEntryInGrid(creatureList, player, NPC_NERUBAR_BROODKEEPER, 135.0f);
```

Comment:
just wondering.
in your first video, it show this AreaTrigger activing 4 spiders.
But your blizzard video, show it activating 2 spiders, 135 yards is not too much?

Reply:
I'm experimenting with yards.

Reply:
I have tried all possible options from 110-130 yards and there is no perfect option.